src/main/java/pages/ItemDetailsPage.java

```diff
@@ -0,0 +1,55 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class ItemDetailsPage extends HomePage {
+    private final By addToCartButton
+            = By.cssSelector("button[id^=add-to-cart]");
+    private final By itemName = By.cssSelector(".inventory_details_name");
+    private final By itemDescription = By.cssSelector(".inventory_details_desc");
+    private final By itemPrice = By.cssSelector(".inventory_details_price");
+    private final By itemNameInBasket = By.cssSelector(".inventory_item_name");
+    private final By itemDescriptionInBasket = By.cssSelector(".inventory_item_desc");
+    private final By itemPriceInBasket = By.cssSelector(".inventory_item_price");
```

Comment:
Seems like these locators belongs to another HTML page which is BasketPage or ShoppingCartPage...
Why you add them here